app/src/pages/projects/ProjectAssets.tsx

```diff
@@ -19,6 +19,7 @@ import Page from "../../components/Page";
 import { ProjectDataContext } from "../../contexts/project/ProjectDataContext";
 import { MassFormatter, MoneyFormatter } from "../../utilities/Formatters";
 import Refresher from "../../components/Refresher";
+import BrandText from "../../components/menu/components/BrandText";
```

Comment:
If this is being used in more than the menu the file should be pulled out the menu components and put in the components folder

Reply:
As with the other comment, I think we need a generic styled folder for these components?